tests/Resources/Page/OCPDashboard/Events/Events.resource

```diff
@@ -0,0 +1,33 @@
+*** Settings ***
+Documentation       Keyword Suites to work with Openshift Events
+
+Library             Collections
+Library             OpenShiftCLI
+
+
+*** Keywords ***
+Get Event Entry From Openshift
+    [Documentation]    Capture openshift event based on namespace
+    ...    and return filtered data
+    [Arguments]    ${namespace}    ${object_kind}    ${expected_event}=2
+    ${data}    Run Keyword    OpenShiftCLI.Get    kind=Event    namespace=${namespace}
+    IF    len(${data}) >= ${expected_event}
+        ${res_data}    Get Data Based On Object Kind    ${data}    ${object_kind}
+    ELSE
+        FAIL    All the Events are not captured
+    END
+    [Return]    ${res_data}
+
+_Get Data Based On Object Kind
```

Comment:
Why the `_` before the Keyword name?

Reply:
There was a proposal from Jorge to have **_** for Private keyword.

Reply:
I don't like it that much but ok :D

Reply:
Neither do i but i think it will be good practice :D